src/main/java/org/codehaus/mojo/taglist/TagListReport.java

```diff
@@ -485,6 +506,7 @@ public List constructSourceDirs()
             {
                 MavenProject reactorProject = (MavenProject) i.next();
 
+                // TODO should this be more like ! "pom".equals(...)
```

Comment:
Can you elaborate this commend? And maybe even fix it?
It might be better to create an issue with a link to this line (see: https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files so we don't lose the correct line, just press 'y').

Reply:
Yes I don't this comment as well. because it's checked the language and not the packaging type...

Reply:
My proposal would be just to ignore projects of `packaging` `pom` and run on all other projects - this is just an optimization which is maybe not completely necessary? The code should be robust enough to not fail if there are no source directories, or if it does not find any files to analyze, so building in limitations like this may not be a good thing?

Reply:
I created the issue https://github.com/mojohaus/taglist-maven-plugin/issues/55